examples/mobile/HomeApp/js/clipping-storage.js

```diff
@@ -0,0 +1,25 @@
+class Storage {
+	constructor() {
+		this.webclips = JSON.parse(localStorage.getItem("webclips")) || [];
```

Comment:
we shouldn't store webclips list in local storage. this is dynamic list which is changed frequently